test/extended/builds/build_pruning.go

```diff
@@ -317,7 +317,7 @@ var _ = g.Describe("[Feature:Builds][pruning] prune builds based on settings in
 		g.It("[Conformance] buildconfigs should not have a default history limit set when created via the legacy api", func() {
 
 			g.By("creating a build config with the legacy api")
-			err := oc.Run("create").Args("-f", legacyBuildConfig, "--raw=/oapi/v1/namespaces/"+oc.Namespace()+"/buildconfigs").Execute()
+			err := oc.Run("create").Args("-f", legacyBuildConfig, "--raw=/api/v1/namespaces/"+oc.Namespace()+"/buildconfigs").Execute()
```

Comment:
Yeah, this change won't work.  We should make a commit deleting tests like this and let people know they're gone.

Reply:
true, api groups. But we can switch to the specific API group and drop legacy paths.